apps/site/lib/site_web/router.ex

```diff
@@ -202,6 +202,8 @@ defmodule SiteWeb.Router do
   scope "/api", SiteWeb do
     pipe_through([:secure, :browser])
     get("/alerts", AlertController, :show_by_routes)
+
+    get("/stop/:id", StopController, :get)
```

Comment:
The `/api` scope's path/module/function naming schemes are starting to get a bit disorganized here... we should consider streamlining that soon, if not now then when we start to add more /api routes.

Reply:
I am curious what you have in mind?  Standardize the module function names, or a better way to organize the routes? (Perhaps putting them into their own file?)

Reply:
Maybe just standardize function names, and/or route names that are more indicative? But that's something we can definitely think about after we have some more endpoints to work with.